nautobot/extras/querysets.py

```diff
@@ -175,7 +175,7 @@ def get_for_object(self, obj):
         my_groups = []
         # TODO(jathan): 3 queries per DynamicGroup instance
         for dynamic_group in eligible_groups.iterator():
-            if obj.pk in dynamic_group.members.values_list("pk", flat=True):
+            if dynamic_group.members.filter(pk=obj.pk).values_list("pk", flat=True).exists():
```

Comment:
I think I asked you this before but I don't remember the answer. :-) Why do we need the values_list call? Can't we just do `dynamic_group.members.filter(pk=obj.pk).exists()`?

Reply:
I'll triple check but I thought it still tries to retrieve all the fields otherwise. Just a query optimization in other words.